src/pages/Loans/LoansOverview/LoansOverview.tsx

```diff
@@ -1,37 +1,42 @@
 import { useTranslation } from 'react-i18next';
 
-import { H1, P, Stack } from '@/component-library';
+import { Stack } from '@/component-library';
 import FullLoadingSpinner from '@/components/FullLoadingSpinner';
 import MainContainer from '@/parts/MainContainer';
 import { useGetLoansData } from '@/utils/hooks/api/loans/use-get-loans-data';
 
 import { LoansInsights, LoansMarkets } from './components';
+import { StyledTitle } from './LoansOverview.style';
 
 const LoansOverview = (): JSX.Element => {
   const { t } = useTranslation();
   const { overview, lendPositions, borrowPositions, assets } = useGetLoansData();
-  const { interestEarnedUSDValue, borrowUSDValue: borrowBalance, loanStatus, supplyUSDValue: supplyBalance } = overview;
+  const {
+    interestEarnedUSDValue,
+    borrowUSDValue: borrowBalance,
+    supplyUSDValue: supplyBalance,
+    earnedRewardsAmount,
+    hasEarnedRewards
+  } = overview;
 
   if (lendPositions === undefined || borrowPositions === undefined || assets === undefined) {
     return <FullLoadingSpinner />;
```

Comment:
Just a question: did you consider with Jay displaying skeletons of cards in `LoanInsights` instead of rendering the `FullLoadingSpinner` here? It could be nicer UX in my opinion as the UI layout will be visible right from the start using skeletons.

Reply:
That something that needs to discussed with Tom. I prefer as well.